CollAction/Frontend/app/global/MobileNavDraw.tsx

```diff
@@ -3,12 +3,29 @@ import renderComponentIf from "./renderComponentIf";
 
 interface IMobileNavDrawProps {
   loginText?: string;
+  isLoggedIn: boolean;
 }
 
 interface IMobileNavDrawState {
   open: boolean;
 }
 
+function AccountManageNav(props) {
+  if (!!props.isLoggedIn) {
+    return <li><a href="/manage">Account</a></li>;
+  }else {
+    return null;
+  }
+ }
+
+ function LoginStateNav(props) {
+  if (!!props.isLoggedIn) {
```

Comment:
Bit unrelated, but you don't need to use the double negatives if you actually declare isLoggedIn as boolean. This is typescript, so best we use typing as much as possible to make the code more robust.